worldify/scripts/spotify_lib.py

```diff
@@ -15,46 +17,65 @@
 
 SPOTIFY_USER_OAUTH = os.getenv("SPOTIFY_USER_OAUTH")
 
-def _get_spotipy_object(user_id):
-    '''Return a spotipy object with authentications'''
-    token = util.prompt_for_user_token(user_id, SCOPE)
-    sp = spotipy.Spotify(auth=token)
-    return sp
+def _get_request_session_object_with_oauth():
```

Comment:
Do we really need to create a new session object every time we want to contact Spotify?